bin/migrate-oats-data/documents/post_launch/alcs_documents_to_inquiry_documents.py

```diff
@@ -0,0 +1,135 @@
+from common import (
+    setup_and_get_logger,
+    BATCH_UPLOAD_SIZE,
+    OATS_ETL_USER,
+)
+from db import inject_conn_pool
+from psycopg2.extras import RealDictCursor
+
+etl_name = "link_inquiry_documents_from_alcs"
+logger = setup_and_get_logger(etl_name)
+
+"""
+    This script connects to postgress version of OATS DB and links data from ALCS documents to ALCS inquiry_document table.
+
+    NOTE:
+    Before performing document import you need to import Inquiries and Inquiry documents.
+"""
```

Comment:
why not to move this description to link_inquiry_documents function documentation